Source/g_game.c

```diff
@@ -123,7 +123,6 @@ boolean         haswolflevels = false;// jff 4/18/98 wolf levels present
 byte            *savebuffer;
 int             autorun = false;      // always running?          // phares
 
-static int      complevel = MBFVERSION;
```

Comment:
This looks like a lot of unrelated changes. Is this necessary?

Reply:
Not required, but this additional `complevel` variable is unnecessary. This code is quite convoluted, so I want to simplify it a little.

Reply:
Okay, fine! 👍